DSharpPlus/DiscordIntents.cs

```diff
@@ -174,16 +174,27 @@ public enum DiscordIntents
 
         /// <summary>
         /// Whether to include scheduled event messages.
-        /// //TODO: reference events
+        /// // TODO: reference events
```

Comment:
oh also, remove this TODO okty

Reply:
Ok, I added the doc even if all the other methods do not have one